Bugzilla/Install/Filesystem.pm

```diff
@@ -81,12 +82,16 @@ EOT
 
 use constant HT_ASSETS_DIR => <<'EOT';
 # Allow access to .css and js files
-<FilesMatch \.(css|js)$>
-  Allow from all
+<FilesMatch state\.json$>
```

Comment:
<Files "state.json"> would probably be simpler. Why is this content considered unsafe?